packages/yoroi-extension/app/Routes.js

```diff
@@ -448,7 +427,9 @@ export function wrapAssets(
     <AssetsWrapper
       {...assetsProps}
     >
-      {children}
+      <Suspense fallback={null}>
```

Comment:
Maybe we can use some sort of loading? 

We have the .gif/svg of the logo, we can check it with the designers team.

Reply:
I thought of it but found out that the **blank white screen** doesn't actually take so long (a fraction of the second on my machine) so adding a loading page in the `fallback` might cause the user to wonder what was happing. Actually, you can try to do it yourself and you will not notice it is not that good! Try to put the `LoadingScreen` as the `fallback` for the `Suspense` and see if it looks good!

Reply:
Yeah, sounds good. Actually I was thinking of users with slow connection on 3g or other poor infrastructure. I think for those it will be better as they will know what's happening instead of a white screen. We won't have that problem because our connection is fast. Wdyt?

Reply:
Nice idea about the skeleton, I think it will make sense! we can add it in the future!